inc/network/includes-network.php

```diff
@@ -0,0 +1,735 @@
+			?>
+	<div class="error"><p><strong><?php _e( 'Warning:' ); ?></strong> <?php _e( 'An existing WordPress network was detected.' ); ?></p></div>
+	<p><?php _e( 'Please complete the configuration steps. To create a new network, you will need to empty or remove the network database tables.' ); ?></p>
+			<?php
+		}
+	}
+
+	$subdir_match          = $subdomain_install ? '' : '([_0-9a-zA-Z-]+/)?';
+	$subdir_replacement_01 = $subdomain_install ? '' : '$1';
+	$subdir_replacement_12 = $subdomain_install ? '$1' : '$2';
+
+	if ( $_POST || ! is_multisite() ) {
+		?>
+		<h3><?php esc_html_e( 'Enabling the Network' ); ?></h3>
+		<p><?php _e( 'Complete the following steps to enable the features for creating a network of sites.' ); ?></p>
+		<div class="notice notice-warning inline"><p>
+		<?php
+		if ( file_exists( $home_path . '.htaccess' ) ) {
+			echo '<strong>' . __( 'Caution:' ) . '</strong> ';
+			printf(
+				/* translators: 1: wp-config.php, 2: .htaccess */
+				__( 'You should back up your existing %1$s and %2$s files.' ),
+				'<code>wp-config.php</code>',
+				'<code>.htaccess</code>'
+			);
+		} elseif ( file_exists( $home_path . 'web.config' ) ) {
+			echo '<strong>' . __( 'Caution:' ) . '</strong> ';
+			printf(
+				/* translators: 1: wp-config.php, 2: web.config */
+				__( 'You should back up your existing %1$s and %2$s files.' ),
+				'<code>wp-config.php</code>',
+				'<code>web.config</code>'
+			);
+		} else {
+			echo '<strong>' . __( 'Caution:' ) . '</strong> ';
+			printf(
+				/* translators: %s: wp-config.php */
+				__( 'You should back up your existing %s file.' ),
+				'<code>wp-config.php</code>'
+			);
+		}
+		?>
+		</p></div>
+		<?php
+	}
+	?>
+	<ol>
+		<li><p id="network-wpconfig-rules-description">
+		<?php
+		printf(
+			/* translators: 1: wp-config.php, 2: Location of wp-config file, 3: Translated version of "That's all, stop editing! Happy publishing." */
+			__( 'Add the following to your %1$s file in %2$s <strong>above</strong> the line reading %3$s:' ),
+			'<code>wp-config.php</code>',
+			'<code>' . $location_of_wp_config . '</code>',
+			/*
+			 * translators: This string should only be translated if wp-config-sample.php is localized.
+			 * You can check the localized release package or
+			 * https://i18n.svn.wordpress.org/<locale code>/branches/<wp version>/dist/wp-config-sample.php
+			 */
+			'<code>/* ' . __( 'That&#8217;s all, stop editing! Happy publishing.' ) . ' */</code>'
+		);
+		?>
+		</p>
+		<p class="configuration-rules-label"><label for="network-wpconfig-rules">
+			<?php
+			printf(
+				/* translators: %s: File name (wp-config.php, .htaccess or web.config). */
+				__( 'Network configuration rules for %s' ),
+				'<code>wp-config.php</code>'
+			);
+			?>
+		</label></p>
+		<textarea id="network-wpconfig-rules" class="code" readonly="readonly" cols="100" rows="7" aria-describedby="network-wpconfig-rules-description">
```

Comment:
Let's make the box bigger so there's no scroll and the whole script can be seen.

```suggestion
		<textarea id="network-wpconfig-rules" class="code" readonly="readonly" cols="100" rows="31" aria-describedby="network-wpconfig-rules-description">
```

Reply:
Chris recommends: Put this into a filter using output buffering